projects/urls.py

```diff
@@ -1,5 +1,7 @@
+# your main urls.py (core/urls.py или config/urls.py)
```

Comment:
The comment is confusing - are you suggesting this file should be imported in the main urls.py? If so, clarify it for new contributors.